File: unittest/scripts/js_dev_api_examples/validation/collection_add.js
var tstRes = myColl.find().sort(['name']).execute();||
print(tstRes.fetchOne().name);|Mike|
print(tstRes.fetchOne().name);|Sakila|
print(tstRes.fetchOne().name);|Susanne|
testSession.dropCollection('test','my_collection');||
testSession.close();||
testSession = null;||

File: unittest/scripts/py_dev_api_examples/working_with_collections/Collection_add.py
# Create a new collection
myColl = db.create_collection('my_collection')

# Insert a document
myColl.add( {'_id': '1', 'name': 'Sakila', 'age': 15 } ).execute()

# Insert several documents at once
myColl.add( [
{'_id': '5', 'name': 'Susanne', 'age': 24 },
{'_id': '6', 'name': 'Mike', 'age': 39 } ] ).execute()
